content/applications/marketing/surveys/create.rst

```diff
@@ -1,224 +1,334 @@
+  two new buttons: :guilabel:`Open Session Manager` and :guilabel:`Close Live Session`.
 
-Once the survey is created and saved, run a test to check for possible errors before finally
-sending it out to the participants by clicking :guilabel:`Test` in the upper left corner of the
-survey template page.
+  Clicking :guilabel:`Open Session Manager` opens another separate tab to the *Session Manager*, and
+  clicking :guilabel:`Close Live Session` closes, and subsequently ends, the live session.
 
-When activated, Odoo redirects the page to a test version of the survey on the front end of the
-website. This page displays how the survey will look to participants. Proceed to run through the
-survey, like a normal participant, to check for errors.
+- :guilabel:`Test`: clicking this button opens a new tab to a test version of the survey, in order
+  for the user to check for errors or inconsistencies, from the point-of-view of a participant.
+  Users can tell if they are in a test version of the survey if there is a blue banner at the top of
+  the screen, reading: `This is a Test Survey --> Edit Survey`.
 
-To return to the survey template form in the backend, simply click the :guilabel:`This is a test
-survey. Edit Survey` link in the blue banner along the top of the page. Once Odoo redirects the
-page to the survey template in the backend, make any further changes, as needed, before officially
-sending the survey out to participants.
+  If the link in the blue banner is clicked, Odoo returns the user to the survey form.
 
-When the survey is ready to be shared with the audience, click the :guilabel:`Start Survey` button
-in the upper-left corner of the survey template form. Then, click :guilabel:`Share`.
+- :guilabel:`Print`: clicking this button opens a new tab to a printable version of the survey that
+  the user can proceed to print for their records.
 
-In the pop-up window, add the survey recipients in the :guilabel:`Recipients` field (for existing
-contacts in the Odoo database) or the :guilabel:`Additional emails` field (for contacts that do not
-want to be listed in the Odoo database). Finally, click :guilabel:`Send`.
+- :guilabel:`Close`: clicking this button closes the survey (i.e. archives it), which is represented
+  by a red :guilabel:`Archived` banner across the top-right corner of the survey form.
 
-As answers are received, check them by clicking the :guilabel:`Answers` smart button on the survey
-template form, or the :guilabel:`See Results` button in the upper left corner. To end the survey,
-click the :guilabel:`Close` button on the survey template form.
+  When this button is clicked, and the survey is closed, a single button appears in the upper-right
+  corner of the survey form, titled: :guilabel:`Reopen`. When :guilabel:`Reopen` is clicked the
+  survey is reopened (i.e. unarchived), and the :guilabel:`Archived` banner is removed from the
```

Comment:
I don't believe unarchived is a real word, even though it totally makes sense in our corporate setting
```suggestion
  survey is reopened, and the :guilabel:`Archived` banner is removed from the
```